data_collection/gazette/spiders/ba_angical.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaAngicalSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2901403"
+    name = "ba_angical"
+    start_urls = ["http://ba.portaldatransparencia.com.br/prefeitura/angical/"]
```

Comment:
They've moved to this URL http://www.acessoinformacao.com.br/ba/angical/diario
Not sure if there's a generic spider for this site already, if not it could be another opportunity to add several cities at one.